requirements.txt

```diff
@@ -1,2 +1 @@
 dwave-ocean-sdk>=4.1
-pandas==1.3.3
```

Comment:
Since Pandas is used here, you definitely want to keep it in the requirements, just adjust the bounds.

Reply:
Got it! Thank you.